### PR TITLE
Feat: Relayer CI 

### PR DESCRIPTION
**Pull request by 0x3bfc**
The main purpose of this PR is to do the following:
- [ ] publish relayer code as npm package.
- [ ] Build and publish Docker images (endpoint and database).
- [ ] Provide clear/documented release process.
- [ ] Enforce coding style in the CI

### CODE REVIEW

**Review by artob — APPROVED** [review]

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
```
Comment (artob) [inline .github/workflows/publish.yml:2]: This seems a misleading copy-paste artifact.

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v2
+      with:
+        node-version: 12
+        cache: 'npm'
+    - uses: JS-DevTools/npm-publish@v1
+      with:
+        token: ${{ secrets.NPM_TOKEN }}
```
Comment (artob) [inline .github/workflows/publish.yml:22]: Files shouldn't end without a trailing newline.

**Review by artob — CHANGES_REQUESTED** [review]
Multiple issues to address here:

- The PR title is not descriptive nor useful, and the PR description is empty.

- The workflows in `node.js.yml` are broken, as per the PR job statuses.

- There are misleading copy-paste artifacts in `publish.yml`.

- The mechanism of operation of `publish.yml`, vis-a-vis `NPM_TOKEN`, is unclear and undocumented. Is this intended to publish NPM releases on every merge to master? How will they be versioned?

**Comment by 0x3bfc** [comment]
> Multiple issues to address here:
> 
> * The PR title is not descriptive nor useful, and the PR description is empty.
> * The workflows in `node.js.yml` are broken, as per the PR job statuses.
> * There are misleading copy-paste artifacts in `publish.yml`.
> * The mechanism of operation of `publish.yml`, vis-a-vis `NPM_TOKEN`, is unclear and undocumented. Is this intended to publish NPM releases on every merge to master? How will they be versioned?

Thanks @artob  .... I'll follow up on these comments. it is still WIP.

**Comment by artob** [comment]
> it is still WIP.

Please make sure to mark it as a Draft PR until it is ready for review

**Pull request closed without merging**